src/terra-dev-site/ss/filler/_PlaceholderPage.ss.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
You could write a base filler mdx file and included it multiple times.

base.mdx
```md
FILLER
```
filler1.mdx
```jsx
import Filler from './base.mdx';

<Filler />
```
filler2.mdx
```jsx
import Filler from './base.mdx';

<Filler />
```
etc

Reply:
Ok done: https://github.com/cerner/terra-dev-site/pull/217/commits/c52489d70cfbf8073bc1839a448f19f3bef40f4d